CHANGES.rst

```diff
@@ -6,6 +6,9 @@ Changes
 Unreleased
 ==========
 
+- Add a quick ``make test`` target for validating that the
```

Comment:
As the `make test` is only interested internally and not from an outside view it should not be mentioned here in my opinion.

Reply:
I've removed that comment from the CHANGES.rst file.